invoices/invoiceregistry_test.go

```diff
@@ -674,3 +693,85 @@ func (p *mockPayload) MultiPath() *record.MPP {
 func (p *mockPayload) CustomRecords() hop.CustomRecordSet {
 	return make(hop.CustomRecordSet)
 }
+
+// TestSettleMpp tests settling of an invoice with multiple partial payments.
+func TestSettleMpp(t *testing.T) {
```

Comment:
Looks like the file changed during review. The above comment meant to be here: "I think it's important to test cases where settlement may fail".

Reply:
There is a case where the partial payment times out. Which other cases do you mean? Of course the total number of possible test cases is large, but I tried to cover at least the ones that usually occur.